project/ws/viewer/src/lib/plugins/quiz/components/assesment-modal/assesment-modal.component.ts

```diff
@@ -148,7 +139,8 @@ export class AssesmentModalComponent implements OnInit {
         this.numCorrectAnswers = res.correct
         this.numIncorrectAnswers = res.inCorrect
         this.numUnanswered = res.blank
-        this.passPercentage = this.assesmentdata.generalData.collectionId === 'lex_auth_0131241730330624000' ? 70 : res.passPercent // NQOCN Course ID
+        this.passPercentage = this.assesmentdata.generalData.collectionId ===
```

Comment:
here you can use   /* tslint:disable-next-line:variable-name */ to disable max length .